aepsych/utils.py

```diff
@@ -238,15 +238,15 @@ def _get_ax_parameters(config):
         "common", "choice_parnames", element_type=str, fallback=[]
     )
     choices = [
-        config.getlist(parname, "choices", element_type=str, fallback=["True", "False"])
+        config.getlist(str(parname), "choices", element_type=str, fallback=["True", "False"])
```

Comment:
Are these string casts actually needed? It shouldn't.

Reply:
That's fine. ax itself is also deprecated so in the future this function will be removed.